tensorflow_addons/metrics/matthews_correlation_coefficient.py

```diff
@@ -42,18 +42,18 @@ class MatthewsCorrelationCoefficient(tf.keras.metrics.Metric):
           ((TP + FP) * (TP + FN) * (TN + FP ) * (TN + FN))^(1/2)
 
     Usage:
-    ```python
-    actuals = tf.constant([[1.0], [1.0], [1.0], [0.0]],
-             dtype=tf.float32)
-    preds = tf.constant([[1.0], [0.0], [1.0], [1.0]],
-             dtype=tf.float32)
-    # Matthews correlation coefficient
-    mcc = MatthewsCorrelationCoefficient(num_classes=1)
-    mcc.update_state(actuals, preds)
-    print('Matthews correlation coefficient is:',
-    mcc.result().numpy())
-    # Matthews correlation coefficient is : -0.33333334
-    ```
+
+    >>> actuals = tf.constant([[1.0], [1.0], [1.0], [0.0]],
+    ... dtype=tf.float32)
+    >>> preds = tf.constant([[1.0], [0.0], [1.0], [1.0]],
+    ... dtype=tf.float32)
+    >>> # Matthews correlation coefficient
+    >>> mcc = MatthewsCorrelationCoefficient(num_classes=1)
```

Comment:
`tfa.metrics.MatthewsCorrelationCoefficient`?

Reply:
I had my doubt about this but since it was working completely fine. So, I left it as it is.